pkgs/by-name/vo/voxinput/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  makeWrapper,
+  testers,
+  libpulseaudio,
+  dotool,
+  voxinput,
+}:
+
+buildGoModule rec {
+  pname = "voxinput";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "richiejp";
+    repo = "VoxInput";
+    tag = "v${version}";
+    hash = "sha256-ykWb5I3cd3DMDVqYrcmOtCKhLpmob7HBXs5Ek5E7/do=";
+  };
+
+  vendorHash = "sha256-OserWlRhKyTvLrYSikNCjdDdTATIcWTfqJi9n4mHVLE=";
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      package = voxinput;
+      command = "voxinput ver";
+      version = "v${version}";
+    };
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    libpulseaudio
+    dotool
+  ];
+
+  # To take advantage of the udev rule something like `services.udev.packages = [ nixpkgs.voxinput ]`
+  # needs to be added to your configuration.nix
+  postInstall = ''
+    mv $out/bin/VoxInput $out/bin/voxinput
```

```suggestion
    [[ -f voxinput ]] || mv $out/bin/VoxInput $out/bin/voxinput
```

Be careful of case-insensitive filesystem

---

```suggestion
    mv $out/bin/VoxInput $out/bin/voxinput_tmp ; mv $out/bin/voxinput_tmp $out/bin/voxinput
```

Or this may be better considering reproducibility?